docs/theory/gas_physics.rst

```diff
@@ -3,4 +3,218 @@
+
+    HP^a - (c_j\dot{V} + d_j)  - (\delta_{discon}(k) - 1-\delta_{dir})M\geq 0 \\
+    \forall (c_j, d_j) \;\; \forall a \in A_{pipes}.
+
+Here :math:`(c_j, d_j)` are the coefficients and constants of the linear equations used to approximate the quadratic equation.
+
+The method with linear inequalities, as represented above, is only valid when each possible unique flow path route in the network has a control valve to compensate non-physical head loss induced by the optimizer. Alternatively a (piece-wise) linear equality constraint between min and max flow rate can be configured for cases where this assumption is invalid, with the general form represented by :eq:`eq:pipe_head_loss3`-:eq:`eq:pipe_head_loss6`.
+
+.. math::
+    :label: eq:gas_pipe_head_loss3
+
+    dH - (\alpha_j \dot{V} + \beta_j) + (\delta^a_{discon} + \delta^a_{dir} + (1 - \delta^a_{line\_seg_{k}}))M\geq 0 \\ \delta^a_{line\_seg_{k}} \in \{ 0, 1 \}, (\alpha_j, \beta_j) \;\; \forall a \in A_{pipes},
+
+.. math::
+    :label: eq:gas_pipe_head_loss4
+
+    dH - (\alpha_j\dot{V} + \beta_j) - (\delta^a_{discon} + \delta^a_{dir} + (1 - \delta^a_{line\_seg_{k}}))M\leq 0 \\  \delta^a_{line\_seg_{k}} \in \{ 0, 1 \}, (\alpha_j, \beta_j) \;\; \forall a \in A_{pipes},
+
+.. math::
+    :label: eq:gas_pipe_head_loss5
+
+    dH + (\alpha_j \dot{V} + \beta_j) - (\delta^a_{discon} + (1 - \delta^a_{dir}) + (1 - \delta^a_{line\_seg_{k}}))M\leq 0 \\ \delta^a_{line\_seg_{k}} \in \{ 0, 1 \}, (\alpha_j, \beta_j) \;\; \forall a \in A_{pipes},
+
+.. math::
+    :label: eq:gas_pipe_head_loss6
+
+    dH + (\alpha_j\dot{V} + \beta_j) + (\delta^a_{discon} + (1-\delta^a_{dir}) + (1 - \delta^a_{line\_seg_{k}}))M\geq 0 \\  \delta^a_{line\_seg_{k}} \in \{ 0, 1 \}, (\alpha_j, \beta_j) \;\; \forall a \in A_{pipes}.
+
+Where :math:`(\alpha_j, \beta_j)` are the coefficients and constants of the linear equations used to approximate the quadratic equation. Variable :math:`\delta^a_{line\_seg_{k}}` reperesents an integer, for pipe :math:`a`, value indicating if a linear line segment :math:`k` is active (value = 1) or not (value = 0). This would imply that if a quadratic curve is represented by 3 linear lines for instance, then only 1 of the linear lines are appplicable at a specific timestep.
+
+
+Node
+~~~~
+
+The node conserves the flow with :eq:`eq:Q_balance` and the energy with :eq:`eq:m_balance`.
+
+The head of all pipes connected to the node must be equal to ensure a hydraulically feasible solution:
+
+.. math::
+    :label: eq:gas_node_head
+
+    H_i = H^a \;\; \forall i \in I^a \;\; a \in A_{nodes},
+
+where :math:`A_{nodes}` is the set of all nodes and :math:`H^a` is the headloss for node :math:`a`.
+
+
+Asset Physics
+-------------
+
+Source
+~~~~~~
+
+The source adds gas mass to the network.
+The volumetric- and mass flow balance is given by :eq:`eq:Q_balance` and :eq:`eq:m_balance` where
+:math:`\dot{V}^a_{consumed}` :math:`\dot{\m}^a_{consumed}` is equal to the (negative) value of the
```

Comment:
- [ ] Should this be volumetric flow rate / mass flow rate?
- [ ] Should the mass flow rate symbol be red?

Reply:
Yes for the first one

for the second point, I personally don't think so, but if your prefer to use red please do so

Reply:
Fixed